### ISSUE TITLE
IVSHMEM broken on windows 7

### ISSUE THREAD

**Issue by Vaporeon** [duncanthrax/scream#46]
I'm sure this is really a bug with Qemu or the IVSHMEM driver, but when drying to use scream on Windows 7, either the OS hangs or I get no audio output. I'm not exactly sure why.

**Comment by martinellimarco** [comment]
Hi @Vaporeon 
I remember we discussed this in #36 but I never wrote a follow up. I tested it on Windows 7 and I confirm that the problem is the IVSHMEM driver, as reported also in the issue I mentioned you the last time.
I see that you wrote a message to that issue also. I hope they'll fix it, unfortunately there is nothing I can do in Scream to work around this.
I suggest you use the network version for now, it does work well, expecially if you have a virtio network between host and guest.

I'm closing this since it's not a Scream issue, sorry.
